### ISSUE TITLE
IE7 Problems

### ISSUE THREAD

**Issue by dnrce** [TracksApp/tracks#959]
_Migrated from the original issue at https://www.assembla.com/spaces/tracks-tickets/tickets/959_

Environment:
Running Tracks 1.7 from the Turnkey Linux, the ISO can be downloaded from the following location: http://www.turnkeylinux.org/tracks

Problem:
There are GUI/Functionality issues with Internet Explorer v.7.0.5730.13

Please reference the attached AVI file to get a better understanding of the issues.

In short, the drag and drop features are not functional and editing a 'Context' topic doesn't work.  Tracks works great within the Firefox browser, but is plagued with functionality issues within IE.

Thanks,
Roy Hernandez
